rllib/agents/dqn/dqn_tf_policy.py

```diff
@@ -383,41 +384,13 @@ def compute_q_values(policy: Policy,
     return value, logits, dist, state
 
 
-def _adjust_nstep(n_step, gamma, obs, actions, rewards, new_obs, dones):
```

Comment:
Can you document all input variables in this method?

Reply:
> Can you document all input variables in this method?

This method is moved to the one in `rllib/evaluation/postprocessing.py` and I think Sven added the doc string for the method.

Reply:
It'd be useful to have documentation and type annotations for those parameters.

Reply:
> It'd be useful to have documentation and type annotations for those parameters.

Yes, the type annotation is included there.

Reply:
see https://github.com/BonsaiAI/ray/pull/93/files#diff-68633652bfaca9ea068b0fbd5210cb01032ee46fd770487b542e8b1722cfc80bR19